src/tests/JIT/opt/LocalMasks/ChangeMatchUse.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Optimize>True</Optimize>
+    <DebugType>None</DebugType>
+    <NoWarn>$(NoWarn);SYSLIB5003</NoWarn>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!-- Needed for CLRTestEnvironmentVariable -->
+    <RequiresProcessIsolation>true</RequiresProcessIsolation>
```

Comment:
This test does not need to be isolated and you can remove the `CLRTestEnvironmentVariable` below too (it's left-over in the other test I linked, from before tiered PGO was enabled by default)